moquette-0.16/broker/pom.xml

```diff
@@ -14,27 +14,26 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <!-- IMPORTANT: netty.version and netty.tcnative.version must be bumped together!
-             Check Netty pom.xm to know the right version of tcnative, for example
-             https://github.com/netty/netty/blob/netty-4.1.77.Final/pom.xml#L531 -->
         <netty.version>4.1.77.Final</netty.version>
+        <!-- Check Netty pom.xm to know the right version of tcnative, for example
+             https://github.com/netty/netty/blob/netty-4.1.73.Final/pom.xml#L531 -->
         <netty.tcnative.version>2.0.52.Final</netty.tcnative.version>
         <paho.version>1.2.5</paho.version>
-        <h2.version>1.4.199</h2.version>
+        <h2.version>2.1.212</h2.version>
```

Comment:
http://www.h2database.com/html/migration-to-v2.html suggests there are breaking changes, would this be an issue for us?

Reply:
No, because H2 isn't enabled, there is no upgrade issue.